packages/create-block-interactive-template/index.js

```diff
@@ -15,6 +15,13 @@ module.exports = {
 		},
 		render: 'file:./render.php',
 		viewScript: 'file:./view.js',
+		customBlockJSON: {
+			example: {
+				attributes: {
+					message: 'Example Interactive',
+				},
+			},
+		},
```

Comment:
So, out of curiosity, what is this `example` thing teaching the developer? How to add attributes?

Reply:
The `example` property allows a preview of the block in the inserter.

## With the property defined:
![example](https://github.com/WordPress/gutenberg/assets/1259027/a5906a9e-6e1e-4c7f-98b3-bf90a7c6c43b)

## Without:
![no_example](https://github.com/WordPress/gutenberg/assets/1259027/f0c4eca4-c203-4c8c-83d1-5852b24e03b5)

Reply:
This does beg the question of the point of that inner `attribute` property. I don't think it's actually being used.

Reply:
Looks like by just having `example` defined, the preview appear. If we were actually using `attributes`, we'd need to add those as well. Change incoming.

Reply:
Ohhhh, nice. 

I completely overlooked that `example` property until now 😄